templates/slots.html

```diff
@@ -60,10 +60,11 @@
       }
       var usp = new URLSearchParams(window.location.search)
       var q = usp.get("q")
+      var t = $("#slots").DataTable(tblOpts)
```

Comment:
**Minor:** This should be placed within the if below.

Reply:
well, not really. This line actually creates the Datatable the first time. The initial draw. Then we check if we had anything as a query parameter and if yes, we redraw the table with the `t.draw()` using the new search parameter(s)